tests/i18n/tests.py

```diff
@@ -1424,8 +1424,17 @@ def test_check_for_language(self):
         self.assertTrue(check_for_language('en'))
         self.assertTrue(check_for_language('en-us'))
         self.assertTrue(check_for_language('en-US'))
+        self.assertTrue(check_for_language('be'))
+        self.assertTrue(check_for_language('be@latin'))
+        self.assertTrue(check_for_language('sr-RS@latin'))
+        self.assertTrue(check_for_language('sr-RS@12345'))
```

Comment:
I've quite intentionally kept it matching anything (same as language or country codes), but we can indeed be more strict.

Anyway there can be more in use, for example ``ca@valencia``: https://github.com/translate/translate/blob/master/translate/lang/data.py#L44